README.md

```diff
@@ -6,6 +6,14 @@ The Chef Management Console is a proprietary addon for the Chef Server. This rep
 
 If you need to file an issue against another Chef project, you can find a list of projects and where to file issues in the [community contributions section](https://docs.chef.io/community_contributions.html#issues-and-bug-reports) of the [Chef docs](https://docs.chef.io).
 
+# Changelog
```

Comment:
I think you're right though, it makes sense to drop this line from the readme, since the file is there and fits convention, so I'll do that.